EngineeringModel/Views/Dialogs/BinaryRelationshipDialog.xaml

```diff
@@ -27,7 +29,7 @@
                         View="Tabs"
                         lc:LayoutControl.AllowHorizontalSizing="True">
             <lc:LayoutGroup Header="Basic" Orientation="Vertical">
-                <items:NameLayoutItem/>
+                <items:NameLayoutItem Visibility="{Binding Converter={StaticResource CdpVersionToVisibilityConverter}, ConverterParameter='Name'}"/>
```

Comment:
i am surprised to see the visibility being used instead of disabling as we discussed?

Reply:
Correct. I tried to implement that, but the way the current solution is structured that would need more time to implement than I have now.
I decided to just implement the current strategy (Visibility).
I will create issues that will take care of the next steps, related to IME, SDK and maybe even WebService.